src/values/UnDeserializableValue.js

```diff
@@ -117,8 +120,14 @@ var SELF = dv.UnDeserializableValue = util.inherit(
 	 * @throws {Error} whenever the function is called (implementation missing).
 	 */
 	toJSON: function() {
-		// TODO
-		throw new Error( 'Not implemented yet.' );
+		return {
+			type: 'undeserializable',
+			data: {
+				targetType: this.getTargetType(),
```

Comment:
This is quite different from how it was before. I do not know how much this matters. What was your reason to change it?